portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -20,9 +20,9 @@ let map;
 let geocoder;
 let marker;
 const google = window.google;
-// Center points to the middle of the United States
-const INITIAL_LAT = 39.8283;
-const INITIAL_LNG = -98.5795;
+// Center points to Minneapolis, Minnesota
+const INITIAL_LAT = 44.9778;
+const INITIAL_LNG = -93.2650;
```

Comment:
Nit: Relying on the comments for additional information of variables is not stable as comments often go out of sync with code. Try to change the variable name too, maybe "MPLS_LAT", "MPLS_LNG".

Reply:
Fixed.